api/proto/teleport/externalaudit/v1/externalaudit.proto

```diff
@@ -0,0 +1,68 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.externalaudit.v1;
+
+import "teleport/header/v1/resourceheader.proto";
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/externalaudit/v1;externalauditv1";
+
+// ExternalAudit contains external audit configuration.
+// It's used only in Teleport Cloud with feature called "bring your own bucket".
+// It contains configuration that allows store audit events and session
+// recordings on customer infra instead of Teleport Cloud.
+message ExternalAudit {
```

Comment:
The name `ExternalAudit` is far too generic for it to mean "configuration for athenaevents and s3sessions using the cloud-specific OIDC integration for aws credentials". Maybe `ExternalCloudAudit`? Is this going to be in a service that's only spun up in Teleport Cloud?